cypress/utils/commands.js

```diff
@@ -46,7 +46,15 @@ Cypress.Commands.overwrite('visit', (orig, url, options) => {
         auth: ADMIN_AUTH,
       };
     }
-    newOptions.qs = { security_tenant: CURRENT_TENANT.defaultTenant };
+    newOptions.qs = {
+      ...newOptions.qs,
+      security_tenant: CURRENT_TENANT.defaultTenant,
+    };
+
+    if (newOptions.excludeTenant) {
+      delete newOptions.qs.security_tenant;
+    }
```

Comment:
It looks we can simply do this?
```suggestion
    if (!newOptions.excludeTenant) {
      newOptions.qs = {
        ...newOptions.qs,
        security_tenant: CURRENT_TENANT.defaultTenant,
      };
    }
```